### PR TITLE
Doc: use closes for features

### PR DESCRIPTION
**Pull request by sims1253**

### CODE REVIEW

File: docs/Getting_Involved/Writing_Good_Commits.rst [hunk 1]
```diff
@@ -116,14 +116,14 @@ Example 1:
 
     Fixes https://github.com/coala-analyzer/coala/issues/269
 
-Example 2:
+Example 2(implemented feature):
 ::
 
-    Readme: Fix typo begining -> beginning
+      Linter: Output command on debug
```
Comment (hypothesist) [inline docs/Getting_Involved/Writing_Good_Commits.rst]: Are the two spaces (before `Linter`) at the beginning intentional?

File: docs/Getting_Involved/Writing_Good_Commits.rst [hunk 2]
```diff
@@ -104,7 +106,7 @@ Example:
 
 Example
 ~~~~~~~
-Example 1:
+Example 1(fixed bug):
```
Comment (sils1297) [inline docs/Getting_Involved/Writing_Good_Commits.rst]: space before the opening paren please

**Comment by sils1297** [comment]
@sims1253 may I refer to https://github.com/coala-analyzer/coala/pull/2305/commits/f87da5dc86ba6eabb8644601473f6b14cde2a159#diff-b691d32d4fae044de7010c519418a0d0L58 ? :)

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala/pull/2305/commits/ea3d8e2c4e78ced9cea46807e1982b2bb709d2d3

**Comment by tushar-rishav** [comment]
@rultor merge

**Pull request merged**